app/api/models/request.py

```diff
@@ -0,0 +1,89 @@
+import uuid
+import json
+from flask import jsonify
+from dbHandler import MyDatabase
+
+db = MyDatabase()
+
+
+class RideRequests:
+    """
+    class for ride requests
+    """
+
+    def __init__(self, ride_id="", passenger="", pickup_point="", destination="",
+                 time="", status=False):
+        self.request_id = str(uuid.uuid1())
+        self.ride_id = ride_id
+        self.passenger = passenger
+        self.pickup_point = pickup_point
+        self.destination = destination
+        self.time = time
+        self.status = status
+        self.errors = []
+
+    def to_json(self):
+        """
+        json representation of the Ride model
+        """
+        return {
+            'Request id': self.request_id,
```

Comment:
instead of saying `Request id`, saying `request_id` would make your work much easier

Reply:
The same idea applies for all json keys